pallets/loans/src/entities/loans.rs

```diff
@@ -359,7 +359,8 @@ impl<T: Config> ActiveLoan<T> {
 				inner.adjust(Adjustment::Increase(amount.balance()?))?
 			}
 			ActivePricing::External(inner) => {
-				inner.adjust(Adjustment::Increase(amount.external()?), Zero::zero())?
+				let when = T::Time::now();
```

Comment:
I guess maturity not being passed already is checked by the `ensure_can_borrw()`?

Reply:
But not in `repay()`